### ISSUE TITLE
DESCRIPTION needs to be updated

### ISSUE THREAD

**Issue by nutterb** [nutterb/redcapAPI#46]
The version number needs to be incremented to 1.3
Stephen Lane needs to be added as the Maintainer
A release date needs to be chosen

**Comment by nutterb** [comment]
Steve, do you think you could try making these changes.  I got a bug report late last week and just patched the code.  It would be nice to get the patched code to CRAN this or next week.

After making the changes run:
```r
devtools::document([local_package_directory])
devtools::check([local_package_directory])
```
Let's stop there and make sure all the checks pass.

Will, do you have any experience using `devtools::release`?  I haven't used it before.  Anything in particular we should look out for?

**Comment by wibeasley** [comment]
That's what I use now, ever since the R-Forge capability (to submit to CRAN) disappeared.  It seems to work.  Once or twice the CRAN humans didn't respond (even though the automated acknowledgement was sent).

**Comment by SteveLane** [comment]
OK, will try and get on to it by the end of the week.

**Comment by SteveLane** [comment]
devtools::document and devtools::check show no errors after update :)

**Comment by nutterb** [comment]
I added a little extra plumbing to prepare the package to be submitted using `devtools::release`.  I also submitted it for checking through `devtools::build_win`.  These were things I was reading about at http://r-pkgs.had.co.nz/release.html.  

You should be getting an e-mail with the results of the `build_win` checks.  Hopefully there won't be any new issues arising from there and then we can release it.

Would you like the honor of choosing a release name, Steve?

**Comment by wibeasley** [comment]
FWIW, I use keep the win-builder code in a separate file that I occasionally run.  It helps me run things more consistently and frequently.  I like that you're using the devel version of of win-builder too.  For me, that's the only dev version of R that REDCapR gets tested against (my local machines are the 'patched' release of R).  I forgot what Travis-CI is.

```r
devtools::build_win(version="R-devel") #CRAN submission policies encourage the development version
```

https://github.com/OuhscBbmc/REDCapR/blob/master/utility_scripts/refresh.R#L29

Also, don't be alarmed if the win-builder check logs are different (as long as they're not worse).  They just use slightly different build options than CRAN, R-Forge, and Travis-CI (and presumably your local machines).  I kinda like the variety of perspectives.

**Comment by SteveLane** [comment]
Sorry for the delay, I was out of service :)
The logs looked fine to me - a couple of notes on the change in maintainer.
I'm happy not to choose a release name - too much pressure!

**Comment by nutterb** [comment]
I think we're ready to push this to CRAN.  I think devtools::release should do it.

I'm thinking "Brunswick Stew" for this release.

**Comment by SteveLane** [comment]
Sure - does the release name go into any of the documentation (there's not one in the DESCRIPTION currently).

**Comment by nutterb** [comment]
There is no place for the release name in the R documentation. It's a Github thing.

After CRAN has accepted the update, there's a 'Releases' tab at the top of the repo's index page. We can tag a new release there, which will make a snapshot of the code and log it in the repo history. This makes sure we can look at the status of the package at any point in its release history.

**Comment by nutterb** [comment]
Oh... The release name isn't really important either.  It's just a way to try to fit in with the cool kids. I've taken to naming the releases after random phrases that pop up in whatever is happening in my life. "Go Royals" was released during the baseball World Series; "Manor Wheat" is the name of the tile pattern I used to refloor a room in my house; "Wasted Bourbon" is how a friend described my effort to make vanilla extract from bourbon and vanilla beans. "Brunswick Stew " is what some friends made for dinner last week (it is traditionally made with squirrel). The more random, the better, in my opinion.
